pkg/tuned/controller.go

```diff
@@ -74,24 +85,38 @@ const (
 )
 
 // Types
-type Bits uint8
+type Bits uint64
 
 type Daemon struct {
-	// reloading is true during the TuneD daemon reload.
-	reloading bool
-	// reloaded is true immediately after the TuneD daemon finished reloading.
-	// and the node Profile k8s object's Status needs to be set for the operator;
-	// it is set to false on successful Profile update.
-	reloaded bool
-	// debugging flag
-	debug bool
-	// bit/set representaton of Profile status conditions to report back via API.
+	// bit/set representation of TuneD options which are remotely controllable.
+	restart Bits
+	// bit/set representation of Profile status conditions to report back via API.
```

Comment:
why are we moving from booleans to a bitset?

Reply:
This is in preparation for the containerized `TuneD`. The way I had the PoC implemented was that we had two bitset files.  One was to control TuneD (remotely from the daemonset), the other one was used to communicate the TuneD status.